.github/workflows/build_and_test_workflow.yml

```diff
@@ -6,9 +6,9 @@ name: Build and test
 # trigger on every commit push and PR for all branches except feature branches and pushes for backport branches
 on:
   push:
-    branches: [ '**', '!feature/**', '!backport/**' ]
+    branches: ['**', '!backport/**']
```

Comment:
we can revert this change that we made to the feature branch.

Reply:
will do